### ISSUE TITLE
Expand construction quest to cover minor constructions

### ISSUE THREAD

**Issue by RubenKelevra** [Helium314/StreetComplete#626]
Currently the "construction is still there?" Quest does not cover streets which have been tagged with `construction=minor`. As it's kinda hard to notice, as it's not shown on the map with any rendering, it would be nice to have SCEE ask about the construction still being there, and if not remove the `construction=minor` tag.

**Proposed Solution**
- Expand the current construction quest to search for ways which are tagged with `construction=minor`.
- UI doesn't need to change
- If user confirms that the construction is gone, remove `construction=minor` tag and the maybe existing checkdate tag.
- If it's still there just add/update the checkdate, like with any other construction quest.

**Comment by mnalis** [comment]
Makes sense to me... Have it been proposed and rejected in upstream StreetComplete @RubenKelevra ?
I think it has reasonable chance to be accepted there, so if that is possible, it should be preferred[^1]

[^1]: to minimize maintenance effort in SCEE, it is preferred that the difference between the two does not grow needlessly; so we add stuff that was rejected (or is almost certain to be rejected if it was proposed due to its quest guidelines) for upstream SC

**Comment by RubenKelevra** [comment]
@mnalis I thought I had quite a long time ago, but looks like I miss remembering, as I can't find a feature request for this.

I'll create a FR.

FR: https://github.com/streetcomplete/StreetComplete/issues/6090

**Comment by westnordost** [comment]
Closing here because this sounds like it can be added to SC.
